docs/en/01_overview.md

```diff
@@ -0,0 +1,198 @@
+The default value is 30, as we are expecting days.
+
+You can determine which JobStatuses are allowed to be cleaned up. The default setting is to clean up "Broken" and "Complete" jobs. All other statuses can be configured with [`CleanupJob.cleanup_statuses`](api:Symbiote\QueuedJobs\Jobs\CleanupJob->cleanup_statuses). You can also define [`CleanupJob.query_limit`](api:Symbiote\QueuedJobs\Jobs\CleanupJob->query_limit) to limit the number of rows queried/deleted by the cleanup job (defaults to 100k).
+
+The default configuration looks like this:
+
+```yml
+Symbiote\QueuedJobs\Jobs\CleanupJob:
+  is_enabled: false
+  query_limit: 100000
+  cleanup_method: "age"
+  cleanup_value: 30
+  cleanup_statuses:
+    - Broken
+    - Complete
+```
+
+## Long-running jobs
+
+If your code is to make use of the 'long' jobs, ie that could take days to process, also install another task
+that processes this queue. Its time of execution can be left a little longer.
+
+```bash
+*/15 * * * * /path/to/silverstripe/vendor/bin/sake dev/tasks/ProcessJobQueueTask queue=large
+```
+
+From your code, add a new job for execution.
+
+```php
+use Symbiote\QueuedJobs\Services\QueuedJobService;
+
+$publish = new PublishItemsJob(21);
+QueuedJobService::singleton()->queueJob($publish);
+```
+
+To schedule a job to be executed at some point in the future, pass a date through with the call to queueJob
+The following will run the publish job in 1 day's time from now.
+
+```php
+use SilverStripe\ORM\FieldType\DBDatetime;
+use Symbiote\QueuedJobs\Services\QueuedJobService;
+
+$publish = new PublishItemsJob(21);
+QueuedJobService::singleton()
+    ->queueJob($publish, DBDatetime::create()->setValue(DBDatetime::now()->getTimestamp() + 86400)->Rfc2822());
+```
+
+## Immediate jobs
+
+Jobs can be declare to run "immediately", rather than being queued.
+What that means in practice depends on your configuration.
+
+By default, these jobs are run on PHP shutdown,
+in the same process which queued the job
+(see [`QueuedJobService.use_shutdown_function`](api:Symbiote\QueuedJobs\Services\QueuedJobService->use_shutdown_function)).
+Code run after shutdown does not affect the response to the requesting client,
+but does continue to consume resources on the server.
+In a worker-based environment such as Apache or Nginx,
+this can have side effects on request processing.
+
+So while this is the easiest setup (zero config or system dependencies),
+there are more robust alternatives:
+
+- [inotifywait](./04_immediate-jobs.md#inotifywait): Works based on watching files. Useful for immediate jobs.
+- [lsyncd](./04_immediate-jobs#lsyncd): Alternative based on watching files.
+
+## Logging and error reporting
+
+Just like any other code in Silverstripe, jobs can create log entries and errors.
+You should use the global `LoggerInterface` singleton
+as outlined in the [framework docs on error handling](https://docs.silverstripe.org/en/4/developer_guides/debugging/error_handling/).
```

Comment:
Update this link and check for any other version-specific doc links

Reply:
Done, didn't find any others